posthog/models/plugin.py

```diff
@@ -70,3 +166,37 @@ class Meta:
     plugin_config: models.ForeignKey = models.ForeignKey("PluginConfig", on_delete=models.CASCADE)
     key: models.CharField = models.CharField(max_length=200)
     value: models.TextField = models.TextField(blank=True, null=True)
+
+
+@receiver(models.signals.post_save, sender=Organization)
+def preinstall_plugins_for_new_organization(sender, instance: Organization, created: bool, **kwargs):
+    if created and not settings.MULTI_TENANCY and not settings.TEST and can_install_plugins(instance):
+        # This in disabled in tests to avoid hitting GitHub API limits
+        for plugin_url in settings.PLUGINS_PREINSTALLED_URLS:
+            Plugin.objects.install(
```

Comment:
Thinking of air-gapped systems, what's the failure scenario here? Skip the plugin or skip the organization?

Reply:
Made it so that each plugin is skipped individually with an stdout message